proto/spire/plugin/agent/workloadattestor/v1/workloadattestor.proto

```diff
@@ -1,30 +1,23 @@
-/** Environment specific plugin to attest a workloads “selector”
-data. a*/
-
 syntax = "proto3";
-package spire.agent.workloadattestor;
-option go_package = "github.com/spiffe/spire/proto/spire/agent/workloadattestor";
-
-import "spire/common/common.proto";
-import "spire/common/plugin/plugin.proto";
+package spire.plugin.agent.workloadattestor.v1;
+option go_package = "github.com/spiffe/spire-plugin-sdk/proto/spire/plugin/agent/workloadattestor/v1";
 
-/** Represents the workload PID.*/
 message AttestRequest {
-    /** Workload PID */
+    // Required. The process ID of the workload to attest.
     int32 pid = 1;
 }
 
-/** Represents a list of selectors resolved for a given PID. */
 message AttestResponse {
-    /** List of selectors */
-    repeated spire.common.Selector selectors = 1;
+    // Optional. Selector values related to the attested workload. The type
+    // of the selector is inferred from the plugin name.
+    repeated string selector_values = 1;
 }
 
 service WorkloadAttestor {
-    /** Returns a list of selectors resolved for a given PID */
+    // Attests the specified workload process. If the process is not attestable
+    // (i.e. not a process the attestor knows anything about), INVALID_ARGUMENT
+    // should be returned. Otherwise the attestor should return zero or more
```

Comment:
Kind of nit-picky, but maybe the correct word is "must" instead of "should", if this is a hard requirement in the contract.
```suggestion
    // must be returned. Otherwise the attestor must return zero or more
```

Reply:
I changed the language to be more declarative than "spec-like".